akka-projection-cassandra/src/test/scala/akka/projection/cassandra/CassandraProjectionSpec.scala

```diff
@@ -148,7 +148,7 @@ class CassandraProjectionSpec
   private val offsetStore = new CassandraOffsetStore(system)
   private val session = CassandraSessionRegistry(system).sessionFor("akka.projection.cassandra.session-config")
   private val repository = new TestRepository(session)
-  private val projectionTestKit = new ProjectionTestKit(testKit)
+  private val projectionTestKit = ProjectionTestKit(testKit)
```

Comment:
👍 
strange that we didn't spot it before.

Reply:
that's why it's so good to write doc code in different package